README.md

```diff
@@ -110,7 +110,11 @@ During fast iterative development, you can also choose to not run eslint/styleli
 This speeds up the build and avoids build failures due to e. g. ill-formatted
 css or other issues:
 
-    $ make LINT=0
+    $ ./build.js -e -s
+
+or
+
+    $ LINT=0 make
```

Comment:
Not a strong opinion, but let's drop this. Cockpit doesn't support it either, and it's nicer to have *one* way of doing things.